projects/driver_controls/src/analog_io.c

```diff
@@ -28,30 +29,31 @@ typedef enum {
 static AnalogIODevice s_analog_devices[ANALOG_IO_DEVICES];
 
 static void prv_input_callback(ADCChannel adc_channel, void *context) {
-  Event e = { 0 };
+  InputEvent event_id = 0;
```

Comment:
This should probably be initialized to something invalid.

Reply:
Done